qt/aqt/utils.py

```diff
@@ -275,7 +275,7 @@ def showCritical(
 
 
 def showInfo(
-    text: str,
```

Comment:
Instead of changing the typing of the utils here, I think it makes more sense to add some ifs to the offending code in webview.py.

Reply:
That makes sense, thanks Abdo.